### ISSUE TITLE
[REQUEST] Allow to auto claim channel Moments, Drops along with the channel Points feature which we have currently

### ISSUE THREAD

**Issue by dpthegrey** [SevenTV/Extension#534]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This feature does not exist in the latest nightly version

- [X] I have checked the latest nightly version

### Feature Description

Currently, 7TV extension Automatically Claims Channel Points, would love to have ability to auto claim Drops, Moments along with channel Points. 

### Other Info

There is an extension on chrome web store which Automatically claims Drops, Moments and Points on twitch named as [Automatic Twitch: Drops, Moments and Points](https://chrome.google.com/webstore/detail/automatic-twitch-drops-mo/kfhgpagdjjoieckminnmigmpeclkdmjm) which has ability to auto claim Drops, Moments and Points all 3 in one extension. Would love to have similar functionality in 7TV.

![image](https://user-images.githubusercontent.com/49444980/233828119-f125c1d5-4b75-454a-921d-2c8ef2b8d7ba.png)
![image](https://user-images.githubusercontent.com/49444980/233828144-c4fa5d65-db30-45dd-bff8-490029d96863.png)

**Comment by AnatoleAM** [comment]
As far as I'm aware, Moments & Drops are already auto-claimed by default on vanilla Twitch, could anyone confirm this? If this is the case then this feature is unnecessary/

**Comment by CopperFig** [comment]
For me, drops are not automatically claimed. 
Not in the chat window and also not on the inventory site.
I have to click something to claim the drop.

searched and didn't find anything about auto claim
https://help.twitch.tv/s/article/mission-based-drops
https://help.twitch.tv/s/article/moments

I m currently using bttv for autoclaim, as described in #493

**Comment by AnatoleAM** [comment]
Well, as far as I know it does not actually matter whether or not you click claim. At the end of the timer, if you had the stream open it still gets claimed.

**Comment by CopperFig** [comment]
can someone else confirm this?

**Comment by CopperFig** [comment]
@AnatoleAM can you show us how this works for you. for me, it's not auto claiming. I have to use a separate extension. 
@dpthegrey can you also show how this works for you.

**Comment by dpthegrey** [comment]
@CopperFig  So, I watched the stream which had drops enabled, with the extension [Automatic Twitch: Drops, Moments and Points](https://chrome.google.com/webstore/detail/automatic-twitch-drops-mo/kfhgpagdjjoieckminnmigmpeclkdmjm) and this is how it looks.

### Chat box of twitch after being able to claim a drop
![image](https://github.com/SevenTV/Extension/assets/49444980/12f15a09-9127-476a-87f0-7be683f8edfb)

### Video overlay of extension showing a drop has been claimed
![image](https://github.com/SevenTV/Extension/assets/49444980/bc92efee-ebc7-472e-b964-4b09b35abdac)

### Inventory showing Drops claimed and In Progress
![image](https://github.com/SevenTV/Extension/assets/49444980/43567d5e-889e-4cbf-8893-174d38ea2af4)

**Comment by dpthegrey** [comment]
a not claimed drop stays there for us to go and click manually

![image](https://github.com/SevenTV/Extension/assets/49444980/2c053335-cd1a-43f3-a71d-f5e8cb770167)

**Comment by CopperFig** [comment]
> a not claimed drop stays there for us to go and click manually

Is this without any extension?
For me, it looks the same if I disable all extensions. It's not claiming anything.

**Comment by dpthegrey** [comment]
> Is this without any extension?

yes, with no extension, the chat notification timer ends and next progress started, then i went to drop inventory and it was showing to manually click and claim the drop, after that i clicked claim and it was claimed.

**Comment by SneakyAzWhat** [comment]
This would be a nice addition, if you happen to miss the manual claiming of the drop, your progress towards the next drop (if there are multiple) won't start until the current one is claimed.

**Comment by lyricalcasanova** [comment]
Bumping this as I still see no option in 7TV for Moments and Drops Auto Claiming only for Channel Points

Would like to not have a dedicated extension for just those if possible.

Thank you all! 👍
